lib/health_check/middleware_health_check.rb

```diff
@@ -0,0 +1,42 @@
+module HealthCheck
+  class MiddlewareHealthcheck
+
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      uri = env['PATH_INFO'.freeze]
+      if uri.include? HealthCheck.uri
+        response_type = uri.sub! '/' + HealthCheck.uri + '.' , ''
```

Comment:
maybe content type extractor could be based on regex sth like this 
`\.(json|xml|html)$`
or Content-Type header ?